tools/specs/Microsoft.Data.SqlClient.nuspec

```
@@ -46,7 +46,7 @@ When using NuGet 3.x this package requires at least version 3.4.</description>
         <dependency id="Microsoft.IdentityModel.JsonWebTokens" version="7.5.0" />
         <dependency id="Microsoft.SqlServer.Server" version="1.0.0"/>
         <dependency id="System.Configuration.ConfigurationManager" version="8.0.0" exclude="Compile" />
-        <dependency id="System.Runtime.Caching" version="8.0.0" exclude="Compile" />
+        <dependency id="Microsoft.Extensions.Caching.Memory" version="6.0.1" exclude="Compile" />
```

Wouldn't we want to use the .NET 8 version of the package for our .NET 8 target?
```suggestion
        <dependency id="Microsoft.Extensions.Caching.Memory" version="8.0.0" exclude="Compile" />
```

---

Updated the package version of Microsoft.Extensions.Caching.Memory to 8.0.0 for target framework .Net8.0.